sklearn/cluster/k_means_.py

```diff
@@ -875,6 +875,7 @@ def fit(self, X, y=None):
         Parameters
         ----------
         X : array-like or sparse matrix, shape=(n_samples, n_features)
+            Coordinates of training data points to cluster
```

Comment:
We usually refer to X as the features of the observations, not the coordinates (they could be colors, grades, blood sugar level… pretty much anything).